### DISCUSSION TITLE
Example with OfflineAudioContext source

### DISCUSSION THREAD

**Discussion by decoder-sh-david** [sapphi-red/web-noise-suppressor#8] [Q&A]
Hi, would it be possible to include an example that works with an offline source like an mp3 file upload?

**Comment by sapphi-red** [comment]
I don't know if it works with OfflineAudioContext. Maybe just swapping [this line](https://github.com/sapphi-red/web-noise-suppressor/blob/d9a73a27253d09a1ca91253670f26f1f30a25253/demo/src/index.ts#L37) might kind a work.
